### DISCUSSION TITLE
How to set sample_size =-1 for insertCSVFromPath

### DISCUSSION THREAD

**Discussion by aszenz** [duckdb/duckdb-wasm#1850] [Q&A]
If detect = true, duckdb fails to insert csv since it doesn't look at all the rows for sniffing column types.
Is their an option to set it to look at all rows?

**Comment by au-re** [comment]
I'm having the same issue, @aszenz did you find a workaround?

**Reply by aszenz** [reply]
Not really,  one can create a table using the raw query syntax instead:

```sql
Create table mycsv as
Select * 
from read_csv_auto('https://example.com/csv', sample_size = -1 )
```

**Reply by aszenz** [reply]
@carlopi sorry for pinging but is this something that can be merged, I can create a PR adding this option, assuming it only needs plumbing through to the final call to read_csv
